src/xdocs/config_javadoc.xml

```diff
@@ -2435,8 +2435,7 @@ class DatabaseConfiguration {}
           For getters and setters for the property <code>allowMissingPropertyJavadoc</code>,
           the methods must match exactly the structures below.
         </p>
-        <div class="wrapper">
-          <pre>
+        <source>
```

Comment:
please move update for MissingJavadocMethod to separate commit of this PR "minor: update html doc for MissingJavadocMethod" or move it to separate PR.

we are strict on rule to not mix unrelated changes in one commit.

Reply:
i made separate commit,i did that in 1st place but checks tests were failing, so i rebase into one.
let me try again pls

Reply:
done